ci/dpcpp.filter

```diff
@@ -0,0 +1,15 @@
+accessor
+atomic
+buffer
+handler
+hierarchical
+id
+invoke
+item
+math_builtin_api
+multi_ptr
+nd_item
+range
+reduction
+specialization_constants
+usm
```

Comment:
I can build all these tests with [yesterday build](https://github.com/intel/llvm/releases/tag/sycl-nightly%2F20211004) except:
buffer
reduction
specialization_constants 
usm

How can I reproduce your results?

Reply:
I've tried it for the [version currently used in CI](https://github.com/KhronosGroup/SYCL-CTS/blob/23d2b9be80fcd08b48e5e4cb73d1efdd186e6b07/.github/workflows/build_ci_containers.yml#L44), i.e., `f5c838cd`. Sounds like we maybe should bump that! :-).

Reply:
Right. I think https://github.com/intel/llvm/pull/4587 fixes build of several tests.

Reply:
Let's update it in a separate change.